File: codeforces/cf1030A_in_search_of_an_easy_problem.rs
// https://codeforces.com/problemset/problem/1030/A
use std::io;

fn main() {
    io::stdin()
        .read_line(&mut String::new())
        .unwrap();

    let mut line = String::new();

    io::stdin()
        .read_line(&mut line)
        .unwrap();

    let words: Vec<i64> =
        line
        .split_whitespace()
        .map(|x| x.parse().unwrap())
        .filter(|x| x != &0)
        .collect();

    if words.len() > 0 {
        println!("HARD");
    } else {
        println!("EASY");
    }

}

File: codeforces/cf1030A_in_search_of_an_easy_problem.cpp
// https://codeforces.com/contest/1030/problem/A

#include<iostream>

using namespace std;

int main() {
    int n, tmp;
    cin >> n;
    bool is_easy = true;
    while (n--) {
        cin >> tmp;
        if (tmp == 1) is_easy = false;
    }
    if (is_easy) cout << "EASY";
    else cout << "HARD";

    return 0;
}
